cmd/di.go

```diff
@@ -182,6 +183,8 @@ func (di *Dependencies) Bootstrap(nodeOptions node.Options) error {
 
 	di.PortPool = port.NewPool()
 
+	go upnp.ReportNetworkGateways()
```

Comment:
Do we want to collect data from everyone  even though they are only consumers? Or is this only really useful when dealing with providers? Just asking if we should really call it here or not.

Reply:
Thanks for pointing that out. I think it's fine to collect from consumers as well. Less logic branching, and anonymous data, so I don't see an issue here.